qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -125,38 +123,24 @@ def test_get(self):
                 }))
 
     def test_put(self):
-        self.set_config_key('config/mgr/mgr/telemetry/contact', '')
-        self.set_config_key('config/mgr/mgr/telemetry/description', '')
-        self.set_config_key('config/mgr/mgr/telemetry/interval', '72')
-        self.set_config_key('config/mgr/mgr/telemetry/leaderboard', 'False')
-        self.set_config_key('config/mgr/mgr/telemetry/organization', '')
-        self.set_config_key('config/mgr/mgr/telemetry/proxy', '')
-        self.set_config_key('config/mgr/mgr/telemetry/url', '')
-        self.set_config_key('config/mgr/mgr/telemetry/last_opt_revision', '2')
-        self.set_config_key('config/mgr/mgr/telemetry/enabled', 'True')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_level', 'critical')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_to_cluster', 'False')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_to_cluster_level', 'info')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_to_file', 'True')
```

Comment:
you want to use `selftest` instead? I'd like test_orchestrator to go away.

Reply:
`selftest` is not listed in mgr modules (`mgr.get('mgr_map')['modules']`), thus it can't be used for this test.

Reply:
I'm using `iostat` instead.